SwanCustomEnvironments/swancustomenvironments/scripts/builders/conda.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# Create environment and install packages
+${BUILDER_VERSION} env create -f ${REQ_PATH} -n ${ENV_NAME} -y
+
+# With a symlink on the user's home, no other customenvs can be created
```

Comment:
What is this symlink for? To detect that the conda environment already exists? How come we need it for conda and not for the other types?

Reply:
Conda has the following option
```
-p PATH, --prefix PATH
                        Full path to environment location (i.e. prefix).
```
However, when using it by itself, we get the following error:
```
mamba env create -p /home/rhenriqu/test_env 
...
EnvironmentFileNotFound: '/eos/home-i02/r/rhenriqu/environment.yml' file not found
```
It is complaining about not getting a requirements file. So, passing one, we get this new error:
```
mamba env create -p /home/rhenriqu/test_env -f local/std_python/requirements.txt 
...
SpecNotFound: Environment with requirements.txt file needs a name
```
This time, we need to provide a name for the environment
```
mamba env create -p /home/rhenriqu/test_env -f local/std_python/requirements.txt -n test_env
usage: mamba create [-h] [-f FILE] [-n ENVIRONMENT | -p PATH] [-C] [-k] [--offline] [--force] [--no-default-packages] [--subdir SUBDIR]
                    [--json] [-v] [-q] [-d] [-y] [--solver {classic,libmamba}]
                    [remote_definition]
mamba create: error: argument -n/--name: not allowed with argument -p/--prefix
```

So, the only option left is to create the environment only specifing name and requirements.
This is the reason why, as mentioned previously, conda environments are created under `/opt/conda/...`
The symlink is made for creating a reference under `/home/$user` so the script above is able to detect it and not allowing the creation of any other customenv

Reply:
For me this works:
```
mamba create -p /root/new_env --file /root/requirements.txt
```
Can you try with `--file` instead of `-f`?

Reply:
No way it worked.. 😮‍💨

Reply:
Changing...

Reply:
Ahh but hold on, the difference is not between `-f` and `--file`. You just used `mamba create` instead of `mamba env create` idk if there's any difference... Gotta do some research